README.md

```diff
@@ -238,71 +269,69 @@ code changes and content changes, they will be caught early on. The Storyblok
+development, preview, and production. The development and preview environments
 contain the preview key to Storyblok. The production environment contains the
-public key to Storyblok. The public key only allows access to the published
-version of content, not the draft version.
+public key to Storyblok, which only allows access to the published
+version of the content, not the draft version.
 
 Each Vercel project is also configured with a webhook that Storyblok uses to
-kick off a preview deploy of the `develop` branch whenever content is published
+kick off a preview deployment of the `develop` branch whenever new content is published
 in Storyblok.
 
 ### Slack
 
 There is a Vercel app for Slack. It is set up to send deployment notifications
-to the `#qwebsite-ops` channel.
+to the `#website-vercel-bot-log` channel.
 
 ### Next.js plus the codebase in this repo
 
 The code in this repo is built on top of the Next.js framework. Next.js has a
 feature called preview mode. When preview mode is turned on (via a cookie), the
-Next.js framework dynamically renders each page; otherwise it serves a static,
-pre-rendered page (except when doing local development and then it always
+Next.js framework dynamically renders each page; otherwise, it serves a static,
+pre-rendered page (except when doing local development, and then it always
 dynamically renders each page).
 
 The code in this repo takes advantage of the Next.js preview mode to integrate
 better with Storyblok and Vercel. When the code detects preview mode, it defaults to passing
-`{"version": "draft"}` to the Storyblok API in order to preview content that has
+`{"version": "draft"}` to the Storyblok API to preview content that has
 been saved but not yet published -- in other words, `draft` content.
 
 There is also code that detects when the request comes from a Storyblok iframe.
 In this case, the code forces the browser to automatically enter into Next.js
 preview mode.
 
-To help the end user distinguish which view of the site they are seeing, the
-codebase defines a visual banner at the top of each page. When the site is in
+**To help the end user distinguish which view of the site they are seeing, the
+codebase defines a visual banner at the top of each page.** When the site is in
 preview mode, the banner turns yellow and displays a message telling the user
 that they can see draft content. When the site is not in preview mode, the
 banner turns gray and tells the user that they can see published content. The
 banner provides a clickable link to switch in and out of Next.js preview mode.
-This switch is disabled when the user is viewing the page via the Storyblok UI, because when they are
-working within Storyblok, they should always see the site in Next.js preview
+This switch is turned off when the user is viewing the page via the Storyblok UI because when they are working within Storyblok, they should always see the site in Next.js preview
```

Comment:
Reflow

```suggestion
This switch is turned off when the user is viewing the page via the Storyblok UI because when
they are working within Storyblok, they should always see the site in Next.js preview
```